py/server/tests/test_table.py

```diff
@@ -624,41 +626,24 @@ def closure_fn() -> str:
         inner_func("param str")
 
     def test_nested_scopes(self):
-        _JExecutionContext = jpy.get_type("io.deephaven.engine.context.ExecutionContext")
-        context = (_JExecutionContext.newBuilder()
-                   .captureQueryCompiler()
-                   .captureQueryLibrary()
-                   .captureQueryScope()
-                   .build())
-
-        def inner_func(p) -> str:
-            openContext = context.open()
-            t = empty_table(1).update("X = p * 10")
-            openContext.close()
-            return t.to_string().split()[2]
-
-        t = empty_table(1).update("X = i").update("TableString = inner_func(X + 10)")
+        with make_user_exec_ctx():
+            def inner_func(p) -> str:
```

Comment:
I feel that it is more clear if the method is not defined within the open context. Only the `update` calls need it.

Reply:
Good idea.